css/style.css

```
@@ -280,3 +282,13 @@ textarea {
   color: #eeeeee;
   padding: 12px;
 }
+
+@media (max-width:650px) {
```

```suggestion
@media (max-width:648px) {
```

since the behavior changes under 648 we shouldn't change for others :)